opentech/static_src/src/javascript/main.js

```diff
@@ -20,6 +21,10 @@ import MobileSearch from './components/mobile-search';
             new MobileSearch($(el), $('.header__menus--mobile'), $('.header__search'), $('.js-search-toggle'));
         });
 
+        $(Tabs.selector()).each((index, el) => {
```

Comment:
I wonder if it is worth including the tabs js and the init code per template in `extrajs`. @todd-dembrey ?